tests/conftest.py

```
@@ -1335,6 +1336,124 @@ def factory(
+                                        ocs_operator_pod_ocp.wait_for_resource(
+                                            condition=constants.STATUS_RUNNING,
+                                            selector=constants.OCS_OPERATOR_LABEL,
+                                            resource_count=1,
+                                            timeout=180,
+                                        )
+                                        log.info("ocs-operator pod is Running")
+
+                                        # Wait for another 120 seconds for the radosnamespace to be Ready
+                                        log.info(
+                                            f"Waiting additional 540 seconds for CephBlockPoolRadosNamespace "
+                                            f"'{radosns_name}' to be Ready"
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)